src/webview_provider/QueryResultTab.ts

```diff
@@ -0,0 +1,128 @@
+import { ViewColumn, WebviewPanel, window } from "vscode";
+import { AltimateRequest } from "../altimate";
+import { DBTTerminal } from "../dbt_client/dbtTerminal";
+import { DBTProjectContainer } from "../manifest/dbtProjectContainer";
+import { QueryManifestService } from "../services/queryManifestService";
+import { SharedStateService } from "../services/sharedStateService";
+import { TelemetryService } from "../telemetry";
+import { getFormattedDateTime, provideSingleton } from "../utils";
+import {
+  AltimateWebviewProvider,
+  HandleCommandProps,
+  SharedStateEventEmitterProps,
+} from "./altimateWebviewProvider";
+import { UsersService } from "../services/usersService";
+
+@provideSingleton(QueryResultTab)
+export class QueryResultTab extends AltimateWebviewProvider {
```

Comment:
this new class can be avoided and can be combined with existing queryResultsPanel.